app/models/manageiq/providers/ibm_power_hmc/infra_manager/event_target_parser.rb

```diff
@@ -39,6 +39,12 @@ def parse
         # have changed (e.g. RMCState, PartitionName, PartitionState etc...)
         # This may be used to perform quick property REST API calls to the HMC
         # instead of querying the full LPAR data.
+        if elems[:type].eql?("VirtualIOServer") && /Virtual.+Mapping/ =~ raw_event[:detail]
+          controller_type = "client #{/SCSI/ =~ raw_event[:detail] ? "scsi" : "vfc"} storage adapter"
```

Comment:
If you just want to do a substring check it is slightly more performance to do a simple `raw_event[:detail].include?("SCSI")` than using a regex